docs/asciidoc/modules/ROOT/pages/ml/genai.adoc

```diff
@@ -102,6 +103,104 @@ RETURN m.title
 |===
 
 
+We can use the `additionalPrompts` config to improve the request, e.g. adding the output of the `apoc.ml.schema`.
+In this way the results might be better, since OpenAI is mainly trained to elaborate questions asked in "human language", rather than via Cypher queries.
```

Comment:
Since OpenAI is mainly trained to elaborate natural language questions asked in, rather than Cypher queries, by using this configuration it is possible to achieve better results.